opendistro-for-elasticsearch/opendistro-build
# [HELM] Elasticsearch master: extra containers

*Description of changes:*
User can define extra containers for Elasticsearch master

Test Results:

* Deploy using the following chart values :
```
elasticsearch:
  master:
    extraInitContainers:
      - name: extra-init-container
        image: busybox
        command: ["/bin/bash", "-c", "env"]

    extraContainers:
      - name: extra-container
        image: busybox
        command: ["/bin/bash", "-c", "sleep infinity & wait"]
```
* Check the containers on the Elasticsearch pod

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
